pages/about.tsx

```diff
@@ -126,18 +125,16 @@ const AboutPage = () => {
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H3>The Program Counter</H3>} anchorKey="stack">
+      <SectionWrapper header={<H3>The Program Counter</H3>} anchorKey="counter">
         <p className="pb-8">
           The Program Counter (PC) encodes which instruction, stored in the
-          code, should be read by the EVM sequentially. The program counter is
-          informed by the previous instruction in sequence, as well as available
-          gas, code address, sender, and recipient. While the program counter
-          usually increments by 1 byte for each opcode, the{' '}
-          <RelativeLink to="#60" title="PUSHx" /> instruction is longer than a
-          single byte, and causes the PC to skip their parameter. The{' '}
-          <RelativeLink to="#56" title="JUMP" /> instruction does not increase
-          the PC's value, instead, it modifies the program counter to a position
-          specified by the top of the stack.{' '}
+          code, should be read by the EVM sequentially. The program counter
+          usually offsets by the code region by one byte, with some exceptions.
```

Comment:
This is not really clear. How about:
```suggestion
          usually is incremented by one byte, to point on the following instruction, with some exceptions.
```

Reply:
Instead: "is usually incremented by one byte, and points to the following instruction, with some exceptions."

Reply:
I don't see how the "and" helps here. I'm giving the reason why the PC is incremented by one byte, and you remove the reason and make it like it's independent?

Reply:
Understood. Would you prefer "pointing to the following instruction" or "to point to the following instruction"? Otherwise the grammar would be incorrect.

Reply:
Second one sounds nice, thanks :-)

Reply:
Done!